### PR TITLE
Update teams-for-vdi.md

### PR DESCRIPTION
**Pull request by baardhermansen**
Modified spaces in table text for a better consistency.
Corrected erroneous markdown for code section.
Added markdown for code where needed.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-for-vdi.md [hunk 1]
```diff
@@ -92,42 +98,46 @@ Use the **CsTeamsMeetingPolicy** cmdlets to control the type of meetings that us
...
+| AllowOutlookAddIn |Determines whether a user can schedule Teams meetings in the Outlook desktop client. | Set to False to prohibit a user from  scheduling Teams meetings in the Outlook desktop client. |
+| AllowParticipantGiveRequestControl|Determines whether participants can request or give control of screen sharing.| Set to False to prohibit the user from giving and requesting control in a meeting. |
+| AllowExternalParticipantGiveRequestControl | Determines whether external participants can request or give control of screen sharing. | Set to False to prohibit an external user from giving, requesting control in a meeting. |
+| AllowPowerPointSharing |Determines whether PowerPoint sharing is allowed in a user’s meetings. |Set to False to prohibit a user from sharing PowerPoint files in a meeting. |
+| AllowWhiteboard | Determines whether whiteboard is allowed in a user’s meetings. | Set to False to prohibit whiteboard in a meeting. |
+| AllowTranscription |Determines whether real-time and/or post-meeting captions and transcriptions are allowed in a user's meetings. | Set to False to prohibit transcription  and captions in a meeting. |
```
Comment (JohanFreelancer9) [inline Teams/teams-for-vdi.md]:
```suggestion
| AllowTranscription |Determines whether real-time and/or post-meeting captions and transcriptions are allowed in a user's meetings. | Set to False to prohibit transcription and captions in a meeting. |
```

**Comment by scanum** [comment]
@LanaChin  PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by baardhermansen** [comment]
> Copy Review - Minor edit (please remove extra space).

Sorry about the delay, been away on business trip.

**Pull request merged**